README.md

```diff
@@ -521,6 +521,7 @@ AWS Repos:
 Community Repos:
 
 * [Contribute](https://github.com/donnemartin/awesome-aws/blob/master/CONTRIBUTING.md)
```

Comment:
We could remove this now that there's an item in this section, also for consistency.